docs/overview/README.md

```diff
@@ -80,6 +85,21 @@ If an ingester process crashes or exits abruptly, all the data that has not yet
 been flushed will be lost. Loki is usually configured to replicate multiple
 replicas (usually 3) of each log to mitigate this risk.
 
+#### Timestamp Ordering
```

Comment:
I think we should avoid having this duplicate documentation?  I think i would just reference the docs/architecture.md file rather than duplicating this here.

Reply:
I'd prefer it the other way around, everyone should know about the timestamp ordering rules, and the architecture guide is meant to have a more narrow audience. I'll remove the duplication though.

Reply:
👍